### PR TITLE
apps/poll/userview: fix issue with other choice validation

### PR DESCRIPTION
**Pull request by khamui**

### CODE REVIEW

**Comment by khamui** [comment]
issue description in meinBerlin repo:
https://github.com/liqd/a4-meinberlin/issues/3737

**Comment by Rineee** [comment]
Cool! But I think we would also need for the multiple choice questions, same error there..

File: adhocracy4/polls/assets/PollQuestion.jsx [hunk 1]
```diff
@@ -63,7 +71,7 @@ export const PollQuestion = (props) => {
                       id={'id_choice-' + choice.id + '-single'}
                       value={choice.id}
                       checked={checked}
-                      onChange={(event) => { handleSingleChange(event) }}
+                      onChange={(event) => { handleSingleChange(event, choice.is_other_choice, checked) }}
```
Comment (Rineee) [inline adhocracy4/polls/assets/PollQuestion.jsx:74]: Do you need the checked there as an argument?

Comment (khamui) [reply adhocracy4/polls/assets/PollQuestion.jsx]: oops, yes was a leftover of something i tried before. thanks for the hint

File: adhocracy4/polls/assets/PollQuestion.jsx [hunk 2]
```diff
@@ -63,8 +75,8 @@ export const PollQuestion = (props) => {
                       id={'id_choice-' + choice.id + '-single'}
                       value={choice.id}
                       checked={checked}
-                      onChange={(event) => { handleSingleChange(event) }}
-                      disabled={!props.question.authenticated || props.question.isReadOnly}
+                      onChange={(event) => { handleSingleChange(event, choice.is_other_choice) }}
+                      disabled={!props.question.authenticated | props.question.isReadOnly}
```
Comment (Rineee) [inline adhocracy4/polls/assets/PollQuestion.jsx:79]: Ah, this still works with the second | gone? Was this on purpose?

**Review by Rineee — APPROVED** [review]
Cool, now it all works! I left one small comment, its rather a question..

**Comment by khamui** [comment]
@Rineee sorry my bad! actually it has nothing to do with XOR (as i said in our call). Just fixed.

**Pull request merged**